install-sysdeps.sh

```diff
@@ -7,4 +7,5 @@ sudo apt-get install -y nodejs imagemagick python-sphinx python-yaml \
   python-tz python-virtualenv python-shelltoolbox python-tornado \
   python-gflags g++ xvfb git firefox
 
-sudo npm install -g jshint@2.1.3 mocha-phantomjs@3.3.1 phantomjs@1.9.2-6
+sudo npm install -g jshint@2.1.3 mocha-phantomjs@3.3.1 phantomjs@1.9.2-6 \
+  react-tools
```

Comment:
I think this breaks the charm since react-tools is not in download cache, or if it is in download cache, will the version match? Should the version be pinned like the other packages?